reference/outlook/preview/Recurrence.md

```diff
@@ -0,0 +1,201 @@
+
+
+# Recurrence
+
+The `recurrence` object provides methods to get and set the recurrence pattern of appointments but only to get the recurrence pattern of meeting requests. It will have a dictionary with the following keys: `seriesTime`, `recurrenceType`, `recurrenceProperties`, and `recurrenceTimeZone` (optional).
+
+##### Requirements
+
+|Requirement|Value|
+|---|---|
+|[Minimum mailbox requirement set version](../tutorial-api-requirement-sets.md)|Preview|
+|[Minimum permission level](https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions)|ReadItem|
+|Applicable Outlook mode|Compose or read|
+
+##### States
+
+|State|Editable?|Viewable?|
+|---|---|---|
```

Comment:
We should try to distinguish between an instance and a series

Reply:
I updated to address this. Please take a look.